src/utils/client.js

```diff
@@ -1,4 +1,4 @@
-import 'isomorphic-fetch';
+import 'unfetch/polyfill';
```

Comment:
Considering that fetch is supported in everything but IE11:

- Using the ponyfill would mean all users use a polyfill, when only IE11 needs it. So strong no on that

- Considering the only real difference is that it doesn't support streaming, and it's only used in IE anyway, is it that big an issue? 

Anyway, only other option is to include Github fetch polyfill, but that is 3.5kb of code weight (gzipped!) for _all_ users to support an edge case in IE11.